app-text/texlive-core/texlive-core-2023-r9.ebuild

```
@@ -459,7 +462,8 @@ src_install() {
 
 	dodir "/usr/bin"
 	for i in ${TEXLIVE_MODULE_BINLINKS} ; do
-		[[ -f ${ED}/usr/bin/${i%:*} ]] || die "Trying to install an invalid BINLINK ${i%:*}. This should not happen. Please file a bug."
+		[[ -f ${ED}/usr/bin/${i%:*} ]] || die \
```

I think we could also

```suggestion
		[[ -f ${ED}/usr/bin/${i%:*} ]]
		    || die "Trying to install an invalid BINLINK ${i%:*}. This should not happen. Please file a bug."
```